src/d3a/constants.py

```diff
@@ -44,3 +44,5 @@
 CN_PROFILE_EXPANSION_DAYS = 7
 
 RUN_IN_REALTIME = False
+
+DOUBLE_SIDED_ALGO = 1  # 1--PAB; 2--PAC
```

Comment:
What do you think about defining this as Enum? In my opinion that would be clearer and safer than relying on the comment.